From a91939101bc142f24ab070d68fd5965f72f375ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Mon, 9 Dec 2019 11:56:29 +0100
Subject: [PATCH 1/1] Docker: Fix default value for
 WEBLATE_AUTH_LDAP_USER_DN_TEMPLATE
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This needs to be none to allow search bind.

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 weblate/settings_docker.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/weblate/settings_docker.py b/weblate/settings_docker.py
index 5a1e3dd54d..d5aa61ddfb 100644
--- a/weblate/settings_docker.py
+++ b/weblate/settings_docker.py
@@ -332,9 +332,7 @@ if "WEBLATE_AUTH_LDAP_SERVER_URI" in os.environ:
     from django_auth_ldap.config import LDAPSearch
 
     AUTH_LDAP_SERVER_URI = os.environ.get("WEBLATE_AUTH_LDAP_SERVER_URI")
-    AUTH_LDAP_USER_DN_TEMPLATE = os.environ.get(
-        "WEBLATE_AUTH_LDAP_USER_DN_TEMPLATE", "cn=%(user)s,o=Example"
-    )
+    AUTH_LDAP_USER_DN_TEMPLATE = os.environ.get("WEBLATE_AUTH_LDAP_USER_DN_TEMPLATE")
     AUTHENTICATION_BACKENDS += ("django_auth_ldap.backend.LDAPBackend",)
     AUTH_LDAP_USER_ATTR_MAP = get_env_map(
         "WEBLATE_AUTH_LDAP_USER_ATTR_MAP", {"full_name": "name", "email": "mail"}
-- 
2.24.0

